ext/random/random.c

```diff
@@ -776,6 +776,39 @@ PHP_FUNCTION(random_int)
 }
 /* }}} */
 
+/* {{{ PHP_GINIT_FUNCTION */
+static PHP_GINIT_FUNCTION(random)
+{
+#if defined(ZTS) && defined(COMPILE_DL_RANDOM)
+	ZEND_TSRMLS_CACHE_UPDATE();
+#endif
```

Comment:
This is dead code, because the extension is always built statically. If we will never lift that restriction, the code should be dropped (or maybe it should be dropped regardless of our plans).

Reply:
I don't think this extension will become optional any time soon, as it also provides functions like `mt_rand()` which users expect to exist.

Reply:
OK. delete this.
I did not understand the life cycle of the extension.
I will be careful from now on.

Reply:
@zeriyoshi, no problem. :)